Yoast/wordpress-seo
# Adds prepare route for the MyYoast connection.

## Summary

This PR can be summarized in the following changelog entry:

* N/A.

## Relevant technical choices:

* Tests are not yet finished. 
* JSON encode/decode has been removed as WP already takes care of this, but needs an extra check if they are needed or not. 

## Test instructions

* This is the route: Go to 'http://[your domain]/wp-admin/admin.php?page=wpseo_myyoast&action=connect'. Go there and see you are redirected to MyYoast
* Check the url and see a unique client id is sent. This looks something like this: `client_id=9740f9cf-608e-4327-8a16-24e3ff6a4c0d` (same but different).
*  There should also be a `redirect_url` in the url. 
* All the unit tests should pass. 

## UI changes

## Quality assurance


Fixes #12131
